ci/release/update-version.sh

```diff
@@ -79,6 +79,7 @@ DEPENDENCIES=(
 for FILE in dependencies.yaml conda/environments/*.yaml; do
   for DEP in "${DEPENDENCIES[@]}"; do
     sed_runner "/- ${DEP}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}\.*/g" ${FILE};
+    sed_runner "/-.* ${DEP}-cu.*==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" ${FILE}
```

Comment:
Here's the new sed command we just updated in cudf:
```suggestion
    sed_runner "/-.* ${DEP}\(-cu[[:digit:]]\{2\}\)\{0,1\}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" "${FILE}"
```

Reply:
thanks for making that easy for me!!

Just to be sure, I just applied that and then tested it like this on my mac:

```shell
git pull origin more-rapids-24.4-updates
cp ./ci/release/update-versions.sh ./ci/release/blegh.sh
git checkout branch-24.04
./ci/release/blegh.sh '24.04.00'
rapids-dependency-file-generator --clean

git diff
```

Confirmed that that generates the same set of changes I see checked in here (i.e. that it does catch the `-cu{CUDA_MAJOR}` things).